### PR TITLE
add watch script

### PR DESCRIPTION
**Pull request by privetkometa**

### CODE REVIEW

**Comment by 7rulnik** [comment]
New version of package: `3.6.17-feature-add-watch-script-e4864b4c`

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "flow": "flow",
     "prebuild": "scripts/check-engines.js",
     "build": "BABEL_ENV=esmodules NODE_ENV=production babel src/ -d ./ --ignore stories,test",
+    "watch": "BABEL_ENV=esmodules NODE_ENV=production babel --watch src/ -d ./ --ignore stories,test",
```
Comment (nezed) [inline package.json]:
```
"watch": "yarn build --watch"
```

Comment (privetkometa) [reply package.json]: а че так можно было?)))

**Comment by 7rulnik** [comment]
New version of package: `3.6.17-feature-add-watch-script-312ba42a`

**Pull request merged**